include/deal.II/base/array_view.h

```diff
@@ -131,6 +131,15 @@ class ArrayView
   ArrayView(const ArrayView<typename std::remove_cv<value_type>::type,
                             MemorySpaceType> &view);
 
+  /**
+   * A constructor that automatically creates a view from a value_type object.
+   *
+   * This implicit conversion constructor is particularly useful when calling
+   * a function that takes an ArrayView object as argument, and passing in
+   * a value_type.
+   */
+  ArrayView(value_type &element);
```

Comment:
Would you be opposed to making this explicit? I am worried that things become confusing otherwise.

Reply:
Done.

Reply:
I had thought about that as well, but the other single-argument constructors aren't `explicit` either.  But you're probably right :-)